app/assets/javascripts/polaris_view_components/autocomplete_controller.js

```diff
@@ -125,7 +131,7 @@ export default class extends Controller {
     }
   }
 
-  checkSelected() {
+  markAsSelected() {
```

Comment:
I chose this name over `checkSelected` which could refer to "checking if something is selected".